deploy.sh

```diff
@@ -165,7 +165,7 @@ for proj in $projects; do # bash only
   ./build.sh
 
   if [[ "$fe_projects" = *"$proj "* ]]; then # project is frontend
-    sentry-release.sh $env $RELEASE
+    bash -x sentry-release.sh $env $RELEASE
```

Comment:
Hey Ben, this should not have been included in commit
same for removing env-config/example.env, which also removed directory and as a result actually [broke auto-deployment](https://github.com/sentry-demos/application-monitoring/commit/2372580b68de72196c06c3ecf05a3111a1ffb8eb)
No big deal, the way to avoid it next time is not to use `git add -A`, very common but a bad practice
I'm gonna revert the 2 unwanted changes tomorrow.
@thinkocapo fyi

Reply:
no worries, fixed it
https://github.com/sentry-demos/application-monitoring/commit/767f43fff47b806d282cf9d8b396e1b78371bff0